tests/exec-tests.sh

```diff
@@ -14,11 +14,13 @@ ADD tests/exec-steps.sh tests/step-* /var/lib/tests/
 
 # Do not use the terminal in tests
 RUN sed -i 's|</dev/tty||g' /var/lib/githooks/install.sh && \\
+    sed -i 's|show_prompt ACCEPT_CHANGES.*|echo "Accepted: \$ACCEPT_CHANGES" # disabled for tests|' /var/lib/githooks/install.sh && \\
```

Comment:
Is this needed? We don't show (GUI) prompts during installation.

Reply:
Needed, because it will install the base-template, where we also need to mock the prompts, right?

Reply:
The install script doesn't have the new `show_prompt` function, so it can't call it, right?

Reply:
Hm.. thats true. But when we use the install script `install.sh` in the tests the base_template for example in a `git clone` afterwards has all `show_prompts` in it. I dont know exactly if that is good?
Especially for `step-103.sh` we need to mock all prompts, hm... I am quite unsure, how I should do it.
I thought I mock everything out in `install.sh` as well as the used `/var/lib/githookts/base-template.sh` for all tests...??

Reply:
General thought:
Shouldnt we mock every possible `show_prompt` no matter where, and in the tests inject the required variables there: In the base-template we have 3 prompts:
```
show_prompt TRUST_ALL_HOOKS "  Do you want to allow running every current and future hooks?" "" "y/N" "Yes" "No"

show_prompt ACCEPT_CHANGES "  Do you you accept the changes?" "(Yes, all, no, disable)" "Y/a/n/d" "Yes" "All" "No" "Disable"

show_prompt EXECUTE_UPDATE "    Would you like to install it now?" "" "Y/n" "Yes" "no"
```
Shouldn't we replace these 3 occurances globaly, and also in the `install.sh`?

Reply:
**Question**: Why does test-103.sh work. I have not replaced `show_prompt TRUST_ALL_HOOKS` in the `install.sh`. Shouldn't pop up the "trust" prompt ?? Confused, sorry...

Reply:
See the latest push, the tests work now.